internal/boxcli/root.go

```diff
@@ -49,6 +52,8 @@ func RootCmd() *cobra.Command {
 		SilenceErrors: true,
 		SilenceUsage:  true,
 	}
+	command.SetOut(os.Stdout)
```

Comment:
maybe I am misunderstanding. 

I thought the idea behind `cmd.OutOrStdout()` was that tests could do `command.SetOut` and then inspect the output. 

If we're overriding here what the tests may set, then it won't work as desired.

Reply:
So `out` is not set by default in cobra. So `cmd.OutOrStdout` returns stdout because out is not set. But `cmd.Print` uses `OutOrStderr` (don't ask me why, I don't understand it) so it prints to stderr.

I'll fix this by removing `cmd.Print` and replacing with `fmt.Fprint(cmd.OutOrStdout()`